kong/route_service_test.go

```diff
@@ -418,7 +418,7 @@ func TestRoutesValidationExpressions(T *testing.T) {
 			name: "valid expression with priority",
 			route: &Route{
 				Expression: String(`lower(http.path) ^= "/prefix/"`),
-				Priority:   Uint64(3382097767104500),
+				Priority:   Uint64((1 << 46) - 1),
```

Comment:
How about defining this as a public const (e.g. `MaximalRoutePriority` so that `go-kong` users can determine the maximal priority value? Then, we could also use this here, in the test, instead of hardcoding.

Reply:
Moved to a const.